styleguide/source/assets/scss/03-organisms/_masthead.scss

```diff
@@ -110,8 +110,12 @@
 
     &__container {
       overflow: hidden;
-      display: grid;
```

Comment:
Masthead template relies on grid display. If this is meant for mobile please wrap in breakpo9int

Reply:
Since we are doing mobile first, I wrapped the `display: grid` in a breakpoint below. So any size above mobile will still be `display: grid`. Is this not the proper way to do this?